legate/core/context.py

```diff
@@ -53,16 +52,13 @@ def __call__(self, *args: Any, **kwargs: Any) -> Any:
         ...
 
 
-def find_last_user_frame(libname: str) -> str:
-    for frame, _ in traceback.walk_stack(inspect.currentframe()):
-        if "__name__" not in frame.f_globals:
-            continue
-        if not any(
-            frame.f_globals["__name__"].startswith(prefix)
-            for prefix in (libname, "legate")
-        ):
-            break
-
+def caller_frameinfo() -> str:
+    frame = inspect.currentframe()
+    for _ in range(2):
+        if frame is not None:
+            frame = frame.f_back
+    if frame is None:
+        return "<unknown>"
     return f"{frame.f_code.co_filename}:{frame.f_lineno}"
```

Comment:
This seems equivalent?
```suggestion
    frame = inspect.currentframe()
    if frame is None or frame.f_back is None or frame.f_back.f_back is None:
        return "<unknown>"
    frame = frame.f_back.f_back
    return f"{frame.f_code.co_filename}:{frame.f_lineno}"
```